playbook/app/pb_kits/playbook/pb_title/title.rb

```diff
@@ -6,9 +6,7 @@ class Title < Playbook::KitBase
       prop :color, type: Playbook::Props::Enum,
                    values: [nil, "default", "light", "lighter", "success", "error", "link"],
                    default: nil
-      prop :size, type: Playbook::Props::Enum,
-                  values: [1, 2, 3, 4],
-                  default: 3
+      prop :size, default: 3
```

Comment:
Should it still have a type?

```suggestion
            prop :size, type: Playbook::Props::Number,
                               default: 3
```

Reply:
I see the prop is an object

Reply:
good catch! This definitely needs a type, I would say keep it Enum so we can have the values array as well, what do you think @markdoeswork ?

Reply:
I am talking to @markdoeswork about this because we would have to have mixed types here. It would be exactly like the React version, allowing a string, number, or object that would have a key as string and the value as string or number.

Reply:
We talked and we would have to create a new prop type just for this one. In this case, we preferred to leave it without the type.